### DISCUSSION TITLE
Not creating mr for latest version

### DISCUSSION THREAD

**Discussion by weihan1394** [renovatebot/renovate#27776] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab

### Please tell us more about your question or problem

I am trying to set up my own self-hosted renovate bot and realize some weird issue where it keeps not creating mr even if there is a minor update.

Example: Currently, my package.json is using `"axios": "^1.6.5"` but the latest version is 1.6.7. There is no mr created.

Is there a configuration to ensure that the minor is updated too?

**Comment by weihan1394** [comment]
I am trying to understand from the below logs. Why the current value is always ^1.6.5 even though it is showing that the current and fixed version is 1.6.7?

```
 "npm": [
           {
             "deps": [
               {
                 "depType": "dependencies",
                 "depName": "axios",
                 "currentValue": "^1.6.5",
                 "datasource": "npm",
                 "prettyDepType": "dependency",
                 "lockedVersion": "1.6.7",
                 "updates": [],
                 "packageName": "axios",
                 "versioning": "npm",
                 "warnings": [],
                 "sourceUrl": "https://github.com/axios/axios",
                 "registryUrl": "https://registry.npmjs.org",
                 "homepage": "https://axios-http.com",
                 "currentVersion": "1.6.7",
                 "fixedVersion": "1.6.7"
               },
```

**Reply by rarkins** [reply, marked as answer]
Are you expecting it to update to ^1.6.7 in package.json? (it won't change anything except your package.json). And if so then why do you use ranges at all instead of pinning to exact versions like 1.6.7 instead of ^1.6.7?

Read https://docs.renovatebot.com/configuration-options/#rangestrategy for how to change default behavior plus https://docs.renovatebot.com/dependency-pinning/

**Reply by weihan1394** [reply]
Thanks for the quick response, fixed it with the range strategy.
